src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/PlacementApplicationEntity.kt

```diff
@@ -153,6 +155,14 @@ enum class PlacementApplicationDecision {
   REJECTED,
   WITHDRAW,
   WITHDRAWN_BY_PP,
+  ;
+
+  fun convertToApi() = when (this) {
```

Comment:
nice

Reply:
I think we could do this with a lot of our enum classes

Reply:
I think we could do something like this and then add a function to do the inverse mapping if required (possibly as an extension function with kotlin enums can implement interfaces. Java is inflexible around enums

```
enum class PlacementApplicationDecision(val ApiPlacementApplicationDecision apiDecision) {
  ACCEPTED(accepted),
  REJECTED(rejected),
  WITHDRAW(withdraw),
  WITHDRAWN_BY_PP(withdrawnByPp),
}
```